src/components/cc-addon-info/cc-addon-info.smart-keycloak.md

```diff
@@ -0,0 +1,77 @@
+---
+kind: '🛠 Addon/<cc-addon-info>'
+title: '💡 Smart (Keycloak)'
+---
+# 💡 Smart `<cc-addon-info smart-mode="keycloak">`
+
+## ℹ️ Details
+
+<table>
+<tr><td><strong>Component    </strong> <td><a href="🛠-addons-cc-addon-info--default-story"><code>cc-addon-info</code></a>
+<tr><td><strong>Selector     </strong> <td><code>cc-addon-info[smart-mode="keycloak"]</code>
+<tr><td><strong>Requires auth</strong> <td>Yes
+</table>
+
+## ⚙️ Params
+
+| Name                          | Type          | Details                                                                                         | Default   |
+| ----------------------------- | ------------- | ----------------------------------------------------------------------------------------------- | --------- |
+| `apiConfig`                   | `ApiConfig`   | Object with API configuration (target host, tokens...)                                          |           |
+| `ownerId`                     | `string`      | UUID prefixed with orga_                                                                        |           |
+| `addonId`                     | `string`      | ID of the add-on                                                                                |           |
+| `appOverviewUrlPattern`       | `string`      | Pattern for the application overview url                                                        |           |
+| `addonDashboardUrlPattern`    | `string`      | Pattern for the addon dashboard url                                                             |           |
+| `scalabilityUrlPattern`       | `string`      | Pattern for the scalability url                                                                 |           |
+| `grafanaLink`                 | `GrafanaLink` | Grafana configuration object (may be disabled in some environments)                             | Optional  |
+| `logsUrlPattern`              | `string`      | Pattern for the logs url (Example : `/organisations/${ownerId}/applications/${appId}/logs`)     |           |
+
+```ts
+interface ApiConfig {
+  API_HOST: string;
+  API_OAUTH_TOKEN: string;
+  API_OAUTH_TOKEN_SECRET: string;
+  OAUTH_CONSUMER_KEY: string;
+  OAUTH_CONSUMER_SECRET: string;
+}
+
+interface GrafanaLink {
```

Comment:
doc: can you add explanation about what are `base` and `console` properties (their purpose, their expected format)